src/components/Home/ListTrending/index.js

```diff
@@ -0,0 +1,41 @@
+import React, { Component } from 'react';
+import { connect } from 'react-redux';
+
+import ItemTrending from './itemTrending';
+import { requestTrending } from '../../../actions';
+
+class ListTrending extends Component {
+  componentDidMount = () => {
+    this.props.listTrendings();
+  }
+
+  renderTrendingList = () => {
+    const { listTrending } = this.props;
+    if (listTrending.length > 0) {
```

Comment:
```suggestion
   return listTrending && listTrending.map((item, index) => <ItemTrending key={index} item={item} />)
```
hoặc có thể dùng cái default state để khỏi phải check length